department-of-veterans-affairs/va.gov-cms
# VAGOV-000: Make alert to single value, and update landing page AX.

QA: 
* Go /node/add/landing_page
* The alert field should be single value, and with a select list (not autocomplete).
